src/sentry/rules/filters/issue_occurrences.py

```diff
@@ -25,4 +28,27 @@ def passes(self, event: GroupEvent, state: EventState) -> bool:
         # This value is slightly delayed due to us batching writes to times_seen. We attempt to work
         # around this by including pending updates from buffers to improve accuracy.
         issue_occurrences: int = event.group.times_seen_with_pending
-        return issue_occurrences >= value
+        return bool(issue_occurrences >= value)
+
+    def passes_activity(self, condition_activity: ConditionActivity) -> bool:
+        try:
+            value = int(self.get_option("value"))
+        except (TypeError, ValueError):
+            return False
+
+        try:
+            group = Group.objects.get_from_cache(id=condition_activity.group_id)
+        except Group.DoesNotExist:
+            return False
+
+        now = timezone.now()
+        if now == group.first_seen:
+            return bool(group.times_seen >= value)
```

Comment:
do we need to cast this?

Reply:
yeah mypy complains if its not casted :(